colossalai/tensor/moe_tensor/moe_info.py

```diff
@@ -0,0 +1,13 @@
+from colossalai.tensor import ProcessGroup
```

Comment:
DON'T use this. This will be deprecated soon.

Reply:
update to ProcessGroupMesh